shared/teams/channel/rows/index.tsx

```diff
@@ -0,0 +1,60 @@
+import * as Types from '../../../constants/types/teams'
+import {TabKey} from '../tabs'
+import {getOrderedBotsArray} from '../../team/rows/helpers'
+
+type HeaderRow = {key: string; type: 'header'}
+type DividerRow = {
+  key: string
+  count: number
+  dividerType: 'requests' | 'invites' | 'members'
+  type: 'divider'
+}
+type TabsRow = {key: string; type: 'tabs'}
+type MemberRow = {key: string; username: string; type: 'member'}
+type BotRow = {key: string; username: string; type: 'bot'} | {key: string; type: 'bot-add'}
+type SettingsRow = {key: string; type: 'settings'}
+type LoadingRow = {key: string; type: 'loading'}
+export type Row = BotRow | DividerRow | HeaderRow | LoadingRow | MemberRow | SettingsRow | TabsRow
+
+const makeRows = (
+  channelInfo: Types.ChannelInfo,
+  teamMembers: Map<string, Types.MemberInfo>,
```

Comment:
We should delay adding this plumbing and row types until the implementation tickets.

Reply:
I kind of prefer some basic scaffolding in place to start.

Reply:
Are we going to get the channel participant data from members in `TeamDetails`? That doesn't seem like the place

Reply:
That's how the chat info panel does it

Reply:
It only does that for `#general`. Existing examples of this aren't necessarily done in the best way.

Reply:
K I'll remove it.